clarifai/datasets/upload/examples/README.md

```diff
@@ -1,31 +0,0 @@
-## Dataset upload from local directory
```

Comment:
Curious, how come we are removing all these?

Reply:
Examples are shifted to https://github.com/Clarifai/examples. So that it doesn't make Pypi build big with images/annotation files